pkg/utils/o11y/controller.go

```diff
@@ -309,7 +309,6 @@ func (h *SuiteController) QuayImagePushDeployment(quayOrg, secret, namespace str
 							Name: secret,
 						},
 					},
-					ServiceAccountName: "pipeline",
```

Comment:
hmm I think this one should stay and be named `constants.DefaultPipelineServiceAccount` - or the one defined [here](https://github.com/redhat-appstudio/e2e-tests/pull/508/files#diff-078e075a7b79ec84560e0884411b92d8d930447435092db2d06e2392464c5146R12), wdyt @srivickynesh ?

Reply:
it would use default service account set by tekton Config, so it should be safe to remove.

Reply:
would it have the secret with quay token linked to it?
https://github.com/redhat-appstudio/e2e-tests/blob/87b4374e47729adda1b5c80492f11a8de5c7a208/tests/o11y/o11y-deployments.go#L32-L36

Reply:
@psturc It would be `constants.DefaultPipelineServiceAccount`